reviewing.md

```diff
@@ -1,5 +1,5 @@
 # How to review conformance results
 
   1. Make sure the list of files matches [the one specified here](https://github.com/cncf/k8s-conformance/blob/master/instructions.md#contents-of-the-pr)
-  2. Look at `version.txt`.  Make sure the `client` and `server` versions match each other to the minor version level.  Make sure this also matches the `vX.Y` subdirectory that the PR is in.
+  2. Look at `version.txt`.  Make sure the `client` and `server` versions match each other to the minor version level.  Make sure this also matches the `vX.Y` subdirectory that the PR is in. Also confirm that Y >= 7 (that is, certification is available for Kubernetes versions 1.7 and higher).
```

Comment:
The client minor version(s) can skew by 1 minor revision. 
So 1.8 clients, and e2es, should properly exercise 1.7 clusters.